hermes/bin/build.rs

```diff
@@ -0,0 +1,4 @@
+//! Build
+fn main() {
+    build_info_build::build_script();
```

Comment:
Just wondering why does this crate needed ?

Reply:
It is mention in here https://docs.rs/build-info/0.0.36/build_info/ so build information can be collected at compile time.